presets/4.5/tune/minchan_2022_race.txt

```diff
@@ -0,0 +1,146 @@
+#$ TITLE: MinChan 2022 Race 6s 5"
+#$ FIRMWARE_VERSION: 4.5
+#$ CATEGORY: TUNE
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: 6S, race, world champion, 5 inch, 5", mck, minchan, kim, 2022, multigp
+#$ AUTHOR: MinChan Kim
+
+#$ PARSER: MARKED
+
+#$ DESCRIPTION:
+#$ DESCRIPTION: <img src="https://user-images.githubusercontent.com/2925027/198938547-053e2f86-5491-4006-9205-c5eadc6a3df1.png" width="220px" style="display: block; float: left; margin-right: 10px; margin-top: 10px;"/>
+#$ DESCRIPTION:
+#$ DESCRIPTION: This is a 6S 5'' racing tune by MinChan Kim (aka MCK) from 2022 MultiGP Champs, where he took first place,
+#$ DESCRIPTION: proving again that he is the fastest racer in the world.
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION:
+#$ DESCRIPTION: Original tune comes from 4.3 (KAACK mod), but 4.4 tuning parameters are identical to 4.3, except Anti Gravity.
```

Comment:
```suggestion
#$ DESCRIPTION: Original tune comes from 4.3 (KAACK mod), but 4.5 tuning parameters are identical to 4.4, except Anti Gravity.
```

Reply:
i'll just remove this description line :)